modules/local/nanoq/main.nf

```diff
@@ -0,0 +1,37 @@
+process NANOQ {
+    tag "${meta.id}"
+    label 'process_low'
+    conda "${moduleDir}/environment.yml"
+
+    container "${workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container
+        ? 'https://depot.galaxyproject.org/singularity/nanoq:0.10.0--h031d066_2'
+        : 'biocontainers/nanoq:0.10.0--h031d066_2'}"
+
+    input:
+    tuple val(meta), path(reads)
+
+    output:
+    tuple val(meta), path("*_report.json"), emit: report
+    tuple val(meta), path("*_stats.json"), emit: stats
+    tuple val(meta), env(median), emit: median_length
+
+    script:
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    """
+    nanoq -i ${reads} -j -r ${prefix}_report.json -s -H -vvv > ${prefix}_stats.json
```

Comment:
output the version, you already too in the stub but not in the main script, and it is not an output channel